index.js

```diff
@@ -491,9 +497,9 @@ export default class Carousel extends Component {
     }
 
     render () {
-        const { sliderWidth, itemWidth, containerCustomStyle, contentContainerCustomStyle, enableMomentum } = this.props;
+        const { sliderWidth, itemWidth, containerCustomStyle, contentContainerCustomStyle, enableMomentum, carouselSideMargin } = this.props;
 
-        const containerSideMargin = (sliderWidth - itemWidth) / 2;
+        const containerSideMargin = carouselSideMargin || this.getContainerSideMargin(sliderWidth, itemWidth);
```

Comment:
@skeie Hey! Sorry for not getting back to you sooner; I was on holidays.
If think that we should remove the method `getContainerSideMargin()` altogether. What about this?
```
const containerSideMargin = carouselSideMargin || (sliderWidth - itemWidth) / 2;
```